products/plone.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Plone
+category: server-app
+tags: python-runtime
+permalink: /plone
+releasePolicyLink: https://plone.org/download/release-schedule
+releaseImage: https://plone.org/download/release-schedule/plone-release-schedule-2025-01-23.png/@@images/image
+changelogTemplate: "https://plone.org/download/releases/__LATEST__"
+eolColumn: Security Support
+eoasColumn: Maintenance Support
+releaseDateColumn: Initial release
+
+customColumns:
+  - property: pythonVersions
+    position: after-release-column
+    label: Python versions
+    description: Supported Python versions range
+
+identifiers:
+  - repology: plone
+
+releases:
+-   releaseCycle: "6.1"
+    pythonVersions: 3.10 - 3.13
+    releaseDate: 2025-02-07
+    eoas: false
+    eol: 2027-12-31
+    latest: "6.1.0"
+    latestReleaseDate: 2025-02-07
+
+-   releaseCycle: "6.0"
+    pythonVersions: 3.9 - 3.13
+    releaseDate: 2022-12-12
+    eoas: 2024-12-31
+    eol: 2027-12-31
+    latest: "6.0.14"
+    latestReleaseDate: 2024-12-20
+
+-   releaseCycle: "5.2"
+    pythonVersions: 2.7, 3.8
+    releaseDate: 2019-07-10
+    eoas: 2023-10-31
+    eol: 2024-10-31
+    latest: "5.2.15"
+    latestReleaseDate: 2024-08-01
+
+-   releaseCycle: "5.1"
+    pythonVersions: 2.7
+    releaseDate: 2018-02-05
+    eoas: 2020-10-02
+    eol: 2022-12-12
+    latest: "5.1.7"
+    latestReleaseDate: 2020-10-02
+
+-   releaseCycle: "5.0"
+    pythonVersions: 2.7
+    releaseDate: 2015-09-27
+    eoas: 2019-02-07
+    eol: 2022-12-12
+    latest: "5.0.10"
+    latestReleaseDate: 2019-02-07
+
+-   releaseCycle: "4.3"
+    pythonVersions: 2.7
+    releaseDate: 2013-04-06
+    eoas: 2020-08-28
+    eol: 2022-12-12
+    latest: "4.3.20"
+    latestReleaseDate: 2020-08-25
+
+---
+
+> [Plone](https://plone.org) is an open-source content management system that is used to create,
+> edit, and manage digital content, like websites, intranets and custom solutions. It comes with
+> over 20 years of growth, optimisations, and refinements. The result is a system trusted by
+> governments, universities, businesses, and other organisations all over the world.
```

Comment:
```suggestion
> [Plone](https://plone.org) is an open-source content management system used to create,
> edit, and manage digital content, like websites, intranets, and custom solutions.
```

We prefer to stay as neutral as possible. The last two sentences does not add much to the product description IMO.